### ISSUE TITLE
Warefare and Weaponry

### ISSUE THREAD

**Issue by prappleizer** [prappleizer/galactia#4]
How is warfare in the galaxy primarily carried out? 

What type of weapons do ships carry, how small or large are ships involved in combat scenarios, how many ships constitutes a skirmish, battle, etc.? 

How does arcano-tech play into all this? 

Spawning this thread to start mulling these questions, as it is both an element of the setting that can give it a distinctly sci-fi vibes, and is also something that in some capacity players will almost certainly take part in. 

### Shields
Basically every "high" sci-fi setting has a concept of ship or personal shields — a metal body floating in space is extremely vulnerable to damage and damage to hull integrity is catastrophic, generally. 

I think our magic-concept provides plenty of motivation for ship shields. Brennan Lee Mulligan has been running Starstruck under the mechanic that a ship has *hull hit points* and *shield hit points*, wherein hull damage is permanent (must be repaired), but shield hit points regenerate themselves on some timescale assuming they are not fully depleted. I like this mechanic generally -- something like shields regenerate on the pilot's turn (or ship's turn) on a 5 or 6 on 1d6. 

### Arcanocanons 

I see one unique possibility of our setting is something that allows a spellcaster to "deposit" a ranged spell into a "cannon" which then launches is at another ship or something. Using the arcanocannon would basically be like combining a "distant spell" type effect and possibly a power boost. 

### Laser Cannons
Some type of light/laser/etc thing makes sense — any ideas on making it cool/unique to our setting?

**Comment by hpthatchfield** [comment]
On the note of shields, I think we also will might want to find an elegant way to rip off dune a little bit. Which is to say, high speed projectiles can be stopped by certain types of wearable shields , but low velocity melee weapons can pierce through, motivating non-ranged combat and keeping closer to traditional D&D mechanics.

**Comment by prappleizer** [comment]
Yep, I already intimated as much (or was planning to) when describing the mechanic background — we definitely need something to keep hand-to-hand combat a viable battle strategy people would choose. Personal shields are a great example. I've been mulling a way to do it without COMPLETELY ripping off dune.

**Comment by hpthatchfield** [comment]
ok, what if we rip off a nice combination of Dune and the Fifth Season: a very common piece of combat arcanotech is the Arcano-Magnetic Abjuring Taurus (called AMATs?) which are commonplace, relatively cheap, easy to transport (like a cylinder you can throw on the ground and self deploys) which generates a deflecting field for high speed projectiles. This means that typical combats will be fought within one or more of these AMAT fields, and long-distance ranged combat isn't really useful beyond the first shot. 

This would have big implications for spaceship combat. Maybe there's a heavy boarding / hacking / laser / magic element to combat in order to deactivate the on-board AMATs of your opponents ships. So it doesn't just turn into "who has the most biggest railguns".

**Comment by prappleizer** [comment]
I like where this is heading. There's this key question of shield geometry — why wouldn't it perfectly surround people when activated and make *ranged* combat useless. 

I think a useful starting point is the shield spell. Costs a reaction, lasts till your next turn, ac up by 5. If that's something spellcasters have relatively easy access to (level 1), then it stands to reason that people would find a way to arcanotech it into a device that can be used. Given the low level of the spell, it wouldn't take much energum. So it stands to reason groups planning to get into tussles would keep one that produces a larger shield that can cover multiple individuals. Now.... in 5e, of course, shield is just as good at protecting against melee attacks. But based on your idea, if we move such larger scale personal shields into an *item*, then nothing stops an enemy from simply.... walking through it / to your other side and attacking you from there. (the *spell* shield protects you from all sides, but an object-created shield would more reasonably affect an area/line. I think to avoid the DUNE reference, we just lower the velocity it takes to move through the shield -- an object created shield stops (mostly) projectiles and melee weapons, but not someone just... walking through it. So it's great if you're pinned down and projectiles are all you're dealing with, but don't really help vs melee in *most* cases. 

I think this is readily extendible to ship combat — shielding the full ship for a battle may use a full energum rod, but is doable. It protects against basically all projectiles, but if another ship launches dudes at you, they land on the hull, and stab it with stabby hull piercing shit, that's not protected by shields at all. 

If we made the smaller shield things an item, it could have values of protection (lines, circles) and be one time use (like potion), or some of which recharge (on a die roll in combat, or automatically after a short rest). 

| Cost       |   Protection       |  Recharge?    |
|----------|-----------------|--------------|
| 50 credits | 10 foot line | no | 
| 100 credits | 15 foot line | no |
| 200 credits | 20 foot line | no | 
| 500 credits | 20 foot line | 6| 
| 1500 credits | 40 foot line | 5 - 6|
| 2500 credits | 20 foot sphere | 5 - 6|  
| 5000 credits | 25 foot sphere | 3 - 6 |

Something like this! And again, anyone can *walk* through these. So there's considerations to be had if you're using them against melee, etc.
